tajo-catalog/tajo-catalog-common/src/main/proto/CatalogProtos.proto

```diff
@@ -407,3 +407,8 @@ message IndexResponse {
   required ReturnState state = 1;
   optional IndexDescProto indexDesc = 2;
 }
+
+message GetPartitionsResponse {
```

Comment:
You seem to move this message to here to use in TajoMasterClientProtocol. If so, rename it like other messages.